src/App.js

```diff
@@ -1,11 +1,24 @@
-
+import React from "react";
+import { BrowserRouter, Route } from "react-router-dom";
+import FetchedPosts from "./components/FetchedPosts";
+import Card from "./components/Card";
+import { routes } from "./constants";
```

Comment:
Please pay attention to constants like these. They should looked as 
const SOME_NAME = {
    SOME_FIELD: 'value'
    ...
    };